README.md

```diff
@@ -12,7 +12,25 @@ Note: if you are looking to include additionally include Experiences functionali
 
 ## Install the SDK
 
-The recommended way to install the Rover SDK is via [Cocoapods](http://cocoapods.org/).
+### SwiftPM
+
+The recommended way to install the Rover Campaigns SDK is via SwiftPM.
+
+In Xcode, in your Project Settings, under Package Dependencies, add a new dependency with the URL of this repository: `https://github.com/rover-platform/rover-campaigns-ios`.
```

Comment:
Same issue as the experiences repo. The GitHub URL is incorrect.

Reply:
Aha, that was copypasta. 😅 thanks